core/services/relay/evm/mercury/v3/reportcodec/report_codec.go

```diff
@@ -74,3 +74,11 @@ func (r *ReportCodec) ObservationTimestampFromReport(report ocrtypes.Report) (ui
 func (r *ReportCodec) Decode(report ocrtypes.Report) (*reporttypes.Report, error) {
 	return reporttypes.Decode(report)
 }
+
+func (r *ReportCodec) BenchmarkPriceFromReport(report ocrtypes.Report) (*big.Int, error) {
+	decoded, err := r.Decode(report)
+	if err != nil {
+		return nil, err
+	}
+	return decoded.BenchmarkPrice, nil
+}
```

Comment:
Seeing 3 copies of this is probably a sign that you'll want to generalize the utils for each report version.